package.json

```diff
@@ -84,6 +84,7 @@
     "fix": "gts fix",
     "pretest": "npm run compile",
     "docs": "compodoc src/",
+    "samples-setup": "cd samples/ && npm link ../ && npm run setup && cd ../",
```

Comment:
this seems like a reasonable addition to the script, presumably so that you could setup the sample environment outside of running tests?

Reply:
Yes that is correct. This is very useful if you want to set up the sample environment in a different project or locally.